schedule/yast/yast2_ncurses_gnome.yaml

```diff
@@ -0,0 +1,50 @@
+---
+name:           yast2_ncurses_gnome
+description:    >
+    Test for yast2 UI, ncurses only. Running on created gnome images
+    which provides both text console for ncurses UI tests as well as
+    the gnome environment for the GUI tests.
+conditional_schedule:
```

Comment:
May I ask you to put `schedule` section higher than `conditional_schedule`? We discussed with @rwx788 some time ago that it is more readable when schedule is displayed first. Probably we can apply this as our convention. WDYT?

Reply:
make sense. fixed.